package.json

```diff
@@ -4,10 +4,11 @@
   "description": "",
   "main": "index.js",
   "repository": {
```

Comment:
You simply can do `"repository": "oleg-koval/ya-skeleton"`

Reply:
wow!